src/launcher/java/org/truffleruby/launcher/Launcher.java

```diff
@@ -131,8 +131,8 @@ public static int runRubyMain(Context.Builder contextBuilder, CommandLineOptions
     public static void printPreRunInformation(boolean isGraal, CommandLineOptions config) {
         if (config.isIrbInsteadOfInputUsed()) {
             RubyLogger.LOGGER.warning(
-                    "By default truffleruby drops into IRB instead of reading stdin as MRI. " +
-                            "(Use '-' to explicitly read from stdin.)");
+                    "by default truffleruby drops into IRB instead of reading stdin as MRI - " +
```

Comment:
Maybe a comma would be less confusing than `-` here given there is a `'-'` after?
I would also move it to one-line, it's just hard to see the full message like this.